core/analyzers/python_ast_extractor.py

```
@@ -0,0 +1,374 @@
+        func = node.func
+        if isinstance(func, ast.Name):
+            name = func.id
+            if name not in seen:
+                seen.add(name)
+                calls.append({"name": name, "qualified_name": name})
+        elif isinstance(func, ast.Attribute):
+            attr = func.attr
+            # Reconstruct the full dotted path for simple cases
+            try:
+                full = ast.unparse(func)
+            except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)